src/core/Table/filters/NumberRangeFilter/NumberRangeFilter.test.tsx

```diff
@@ -0,0 +1,143 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import { fireEvent, render, screen } from '@testing-library/react';
+import React from 'react';
+import { HeaderGroup } from 'react-table';
+import { NumberRangeFilter, NumberRangeFilterProps } from './NumberRangeFilter';
+
+/* eslint-disable @typescript-eslint/no-explicit-any */
+const renderComponent = (
+  initialProps?: Partial<NumberRangeFilterProps<any>>,
+) => {
+  const props = {
+    column: {} as HeaderGroup,
+    setFilter: jest.fn(),
+    clearFilter: jest.fn(),
+    close: jest.fn(),
+    ...initialProps,
+  } as NumberRangeFilterProps<any>;
+  return render(<NumberRangeFilter {...props} />);
+};
+/* eslint-enable @typescript-eslint/no-explicit-any */
+
+it('should render correctly', () => {
+  const { container } = renderComponent();
+
+  const labeledInputs = container.querySelectorAll(
+    '.iui-input-container.iui-inline',
+  );
+  expect(labeledInputs.length).toBe(2);
+
+  expect(labeledInputs[0].querySelector('.iui-label')?.textContent).toEqual(
+    'From',
+  );
+  expect(labeledInputs[1].querySelector('.iui-label')?.textContent).toEqual(
+    'To',
+  );
+
+  screen.getByText('Filter');
```

Comment:
I know in more cases we took getByText from render than from screen.. Not sure, whats the diff, but you would not need to import it :D

Reply:
I saw recommendations to use it from `screen` instead destructing from `render` result.